API.html

```diff
@@ -1267,6 +1267,12 @@
     Boolean flag for optionally expanding output with additional whitespace
     for readability. Results in larger but more readable GeoJSON responses.
     </dd>
+    <dt>empty_fields</dt>
```

Comment:
I'd use a name a little bit more descriptive, something like "skip_empty_fields" (default would be true in this case).